### DISCUSSION TITLE
Revising the Enterprise page on WordPress.org

### DISCUSSION THREAD

**Discussion by sereedmedia** [WordPress/Marketing-Team#244] [Ideas]
## Background 
The [WordPress.org Enterprise page](https://wordpress.org/enterprise/) was a roughly two-year project that was published in late 2019. (It was first indexed by the Wayback Machine [Dec 25, 2019](https://web.archive.org/web/20191225144707/https://wordpress.org/enterprise/).)

The page was the result of the efforts of the [Enterprise Growth Council](https://make.wordpress.org/meta/2018/02/01/enterprise-growth-council-meeting-january/), which was established (along with the Consumer Growth Council) in early 2018 after a call-to-marketing-action from Matt Mullenweg in [Dec 2016](https://ma.tt/2016/12/wp-growth-council/). Despite being disbanded at the end of 2018, the members of the Enterprise Growth Council carried on the effort to complete and publish the Enterprise page. 

## Where we are now

The Enterprise page has been brought into the new WordPress.org design schema and can use blocks (which does indeed make content development, design, and iteration much simpler from a technical standpoint!). 

There is also an ongoing discussion happening at #WCEU 2023 around WordPress for Enterprise.

## Ticket purpose
We now have the opportunity to discuss the Enterprise page from a marketing perspective, including defining/refining the intended audience, the goals of the page, and the subsequent Calls to Actions.

This discussion ticket is meant to be a space to house the thoughts and opinions of folks coming from a Marketing perspective AND of folks coming from the WordPress Enterprise space. This page is for Enterprise and should continue to take into consideration the ever-evolving needs of Enterprise-level WordPress usage.

This discussion ticket is also meant to be an updated continuation of/merge into the conversation the [WordPress for Corporate Sized Entities #22](https://github.com/WordPress/Marcomms-Rolling-tasks/issues/22) discussion and efforts from 2021. Hopefully we can use and build off of the content and analytical effort that went into this document from WCEU 2019: [WordPress For Enterprise Buyer Persona](https://docs.google.com/document/d/19HzURxeG-Y8vV5MZqmOUANyc1k4QDADvZDMnQjszFMw/edit#heading=h.nwa9lnlw5yv5
).

<<===================>>

**TL;DR**  If you have any thoughts on what should be on the WordPress.org Enterprise page, can provide insight into the needs or concerns of Enterprise-level WordPress, or have other comments/questions/concerns/ideas related to https://wordpress.org/enterprise/, please leave them below!

<<===================>>

**Comment by mrstrande** [comment]
I would love to discuss this, since it lies very close to my heat and objective for a stronger future WordPress. To be taken serious as an enterprise CMS is in some countries like Denmark is quite important since market shares for more serious solutions often goes to competing CMS and webshop systems. It's hurting the growth of the local market for WordPress and I want to contribute to a change.

Being new to the contribution team, it would be helpful to discuss this at WCEU 2023.
Would a small verbal workshop be a possibility? 

Cheers!

**Reply by sereedmedia** [reply]
@mrstrande you can chat about it all you want, but the actual work will take place here in the GitHub, so if you have thoughts to share, this is where to share them!!

**Comment by sereedmedia** [comment]
For historical reference, this issue has been discusses before: In this (https://github.com/WordPress/Marcomms-Rolling-tasks/issues/22) soon-to-be-closed old GitHub issue, [this Google Doc](https://docs.google.com/document/d/19HzURxeG-Y8vV5MZqmOUANyc1k4QDADvZDMnQjszFMw/edit#heading=h.nwa9lnlw5yv5) was created at a Contributor Day at WCEU (2019?) and includes buyer persons for Enterprise clients. 

> The WordPress marketing team gathered a group of agency owners and marketers at the WCEU Contributors day, to outline the main buyer persona and gather content ideas.
> 
> 1: Objective: Help agencies persuade decision makers in corporate entities of the benefits of WordPress over other systems.
> Working Document: [https://docs.google.com/document/d/19HzURxeG-Y8vV5MZqmOUANyc1k4QDADvZDMnQjszFMw/edit#heading=h.nwa9lnlw5yv5](https://docs.google.com/document/d/19HzURxeG-Y8vV5MZqmOUANyc1k4QDADvZDMnQjszFMw/edit#heading=h.nwa9lnlw5yv5)
> 
> 2: Done =
> -Buyer Persona CMO outlined
> -Buyer persona CTO outlined
> -Content needs for both outlined
> 
> 3: Original Task-Lead: @yvettesonneveld
> 
> Timeline
> 22 April 2020
> Yvette to review latest position

**Comment by mrstrande** [comment]
Great @sereedmedia , I'm new so very sorry if I'm not very aligned with the work-methods just yet.

Let's say that an enterprise-sized client sees https://wordpress.org/enterprise/.
The page consists of:

- Menu to other pages
- Banner with simple text
- Logos of 10 large enterprises that use WordPress
- Text area consisting of one centered text and two columns and two rows of text
- A one-column content area telling about "extensibility,", "security," and "open source"
- A big "Get WordPress" text area
- Footer area consisting of 3 columns: case studies (in PDF form), white paper references (in PDF form), and tutorials (links to Vimeo)
- Footer menu area (global as of https://wordpress.org/)

Overall, the impression is extremely basic. No wow factor, just a bit of content without depth.

Talking to a couple of people, they basically agree that people might most likely just open it and close it very quickly after, without finding what they were looking for. If people are thinking of using WordPress for enterprise, this page might, in fact, just turn them away.

The most important thing that we are missing are case studies. There are so many good case studies. Each case needs to be described in depth and in a way that gives enough information to see that WordPress is a great option for the enterprise.

Furthermore, we need another call-to-action rather than downloading. Here might be an opportunity to perhaps lead the way to WordPress VIP.

Overall, I can deliver some simple wireframes if needed for the ideas for the better landing page, but being new here, I don't know what the next good step would be.

Thanks in advance.

**Comment by sereedmedia** [comment]
The redesign conversation is happening here: https://github.com/orgs/WordPress/projects/40/views/4?pane=issue&itemId=25781101
